src/services/fetchtodos.js

```diff
@@ -0,0 +1,16 @@
+import { checkError, client } from './client';
+
+export async function fetchTodos() {
+  const data = await client.from('todos').select('*');
+  return checkError(data);
+}
```

Comment:
We can also use an orderBy function here to order by Id so that the list of todos does not change.